docs-chef-io/content/inspec/_index.md

```diff
@@ -52,6 +52,6 @@ InSpec to target applications and services running on AWS and Azure.
 
 ### Resources
 
-Chef InSpec has 80+ [resources](/inspec/resources/) ready use--apache to zfs pool.
+Chef InSpec has around 500 [resources](/inspec/resources/) ready use--apache to zfs pool.
```

Comment:
```suggestion
Chef InSpec has 500+ [resources](/inspec/resources/) ready use--apache to zfs pool.
```

Reply:
The command `ls | wc -l` shows 500 files in the docs-chef-io/content/inspec/resources directory. If we use `500+` then we won't need to change it until inspec hits 600 resources.